dist/templates/ovnkube-zone-controller.yaml.j2

```diff
@@ -0,0 +1,412 @@
+---
+# ovnkube-zone-controller
+# daemonset version 3
+# starts zone controller daemons - ovn dbs, ovn-northd and ovnkube-network-controller-manager containers
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-zone-controller
```

Comment:
didn't we want to use `ovnkube-controller` ? we didn't want to use any ovnkube-network-controller-manager or ovnkube-zone-controller specific names and wanted to keep it as generic as possible and agnostic to future changes: https://docs.google.com/presentation/d/10qPTbm4gYdKG0kRu7KI8X-dFuR408onX9ii5jYa6YjQ/edit#slide=id.g22a276f021c_1_0

Reply:
ah nvm! this is multiple node per zone setup: https://docs.google.com/presentation/d/10qPTbm4gYdKG0kRu7KI8X-dFuR408onX9ii5jYa6YjQ/edit#slide=id.g23da337f264_0_0